src/components/LinterMessage/index.tsx

```diff
@@ -32,20 +33,24 @@ const renderDescription = (description: LinterMessage['description']) => {
     }
 
     allLines.push(
-      // Intercept and replace URLs with JSX links.
-      line.split(/(\s+)/).reduce((allParts: React.ReactNode[], part) => {
-        if (urlPattern.test(part)) {
-          allParts.push(
-            <Alert.Link key={`link:${part}`} href={part}>
-              {part}
-            </Alert.Link>,
-          );
-        } else {
-          allParts.push(decodeHtmlEntities(part));
-        }
+      line
+        // Replace anchor tags with just their URLs.
+        .replace(linkPattern, '$1')
```

Comment:
Why not using DOMPurify (already used in this project) to remove all HTML tags (given that we don't intend to render HTML)? The regexp seems fragile if, for instance, new attributes are added (e.g., `target` and `rel`).

Reply:
It seems unlikely to me that this html would change but I could try to see if dompurify is easy enough to work with. It probably won't give me the `href` but maybe that's OK.

Reply:
oh wait, right. it won't work well if the `href` value is different than the link content.